subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/versionupgrade/CreatePullRequestTask.java

```diff
@@ -123,4 +125,34 @@ public void setDryRun(boolean dryRun) {
     public boolean isDryRun() {
         return dryRun;
     }
+
+    /**
+     * Description of pull request for downstream repositories.
+     */
```

Comment:
Thanks for updating the docs! "Description of pull request" should be enough, this task can be used in different contexts than downstream repos.